### PR TITLE
member-settings for admin activated on option hold

### PR DESCRIPTION
**Pull request by DashDeipayan**
### What is the change?

Provide a small description of what did you change and provide the reference to the issue ticket.

### Is it bug?

- Steps to repro
- Expected
- Actual

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [x] Safari
- [ ] Firefox

### Before / After Change Screenshots

> For visual or interaction changes. Can be video / screenshot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +21,19 @@
     "axios": "^0.21.1",
     "cross-fetch": "^3.1.4",
     "framer-motion": "^3.2.1",
-    "next": "^10.0.4",
+    "next": "^12.0.7",
```
Comment (swarajpure) [inline package.json:24]: Woah, wait, why is your PR updating nextjs version?

File: package.json [hunk 2]
```diff
@@ -21,17 +21,19 @@
...
+    "npm": "^8.3.0",
```
Comment (swarajpure) [inline package.json:25]: Looks like you've done `npm install npm` 😂

File: package.json (hunk 1 above)
Comment (DashDeipayan) [reply package.json:24]: I have no idea. Don't even know if it's bad or good.

File: package.json (hunk 2 above)
Comment (DashDeipayan) [reply package.json:25]: No idea...didn't do that for sure 🤣

File: package.json (hunk 1 above)
Comment (ankushdharkar) [reply package.json:24]: ❌  Let's not be doing major upgrades like this. This is dangerous.

Good catch @swarajpure

**Pull request closed without merging**